lte/gateway/fabfile.py

```diff
@@ -247,7 +247,7 @@ def federated_integ_test(
 ):
 
     if orc8r_on_vagrant:
-        # Modify Vagrantfile to allow access to Orc8r running inside Vagrant
+        # Modify Vagrantfile to allow access to orc8r running inside Vagrant
```

Comment:
I believe "Orc8r" is the convention for the way to write the component, see e.g. https://github.com/magma/magma/wiki/Contributing-Documentation#styling-and-writing-conventions in the documentation guidelines. Is there a reason to change it here?

Reply:
I changed this back